docs/book/src/clusterctl/configuration.md

```diff
@@ -101,7 +101,7 @@ You may want to migrate to a user-managed cert-manager further down the line, af
 
 `clusterctl` looks for the label `clusterctl.cluster.x-k8s.io/core=cert-manager` on all api resources in the `cert-manager` namespace. If it finds the label, `clusterctl` will manage the cert-manager deployment. You can list all the resources with that label by running:
 ```bash
-kubectl get all -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
+kubectl get mutatingwebhookconfiguration,validatingwebhookconfiguration,clusterrole,clusterrolebinding,crds,all -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
```

Comment:
There's a number of resources this doesn't cover - e.g. endpoints, but I'm not sure if we have a definitive list of what resources are required for changing management.

I think [this is the full command to actually list every single resource with the specified label](https://github.com/kubernetes/kubectl/issues/151#issuecomment-402003022):

```
kubectl api-resources --verbs=list --namespaced -o name | xargs -n 1 kubectl get --show-kind --ignore-not-found --selector=clusterctl.cluster.x-k8s.io/core=cert-manager -A
```

It's complicated, and slow to run besides, but this is an unusual use case. 

Otherwise the ideal thing to do is to replicate the call that clusterctl does to see if there is a clusterctl managed cert manager on the cluster - https://github.com/kubernetes-sigs/cluster-api/blob/5c38bf731e20be057d4e92f2d4ebbc89d966a31d/cmd/clusterctl/client/cluster/cert_manager.go#L205

That logic is pretty complicated.

Reply:
I think you need to remove `--namespaced` to get **all** api-resources 

```
$ kubectl api-resources --verbs=list --namespaced -o name | wc -l
53
$ kubectl api-resources --verbs=list -o name | wc -l
75
```

Reply:
Thanks for catching that! This convo should be ignored in favor of https://github.com/kubernetes-sigs/cluster-api/pull/7361#discussion_r989910447 which is about the same thing (but without my copy-paste mistake 😆)